providers/aws/services/ec2/ec2_elastic_ip_shodan/ec2_elastic_ip_shodan.py

```diff
@@ -0,0 +1,46 @@
+import shodan
+
+from config.config import get_config_var
+from lib.check.models import Check, Check_Report
+from lib.logger import logger
+from providers.aws.services.ec2.ec2_client import ec2_client
+
+
+class ec2_elastic_ip_shodan(Check):
+    def execute(self):
+        findings = []
```

Comment:
Please, include the `try-catch` for all the check code.

Reply:
Don't think is needed.